### DISCUSSION TITLE
go how to convert lua structure

### DISCUSSION THREAD

**Discussion by wangyufengx** [Kong/kong#9089] [Help]
lua structure
`
policy = {

    type = "string",

    default = "cluster",

    one_of = {

      "local",

      "cluster",

      "redis",

    },

  }
`

I want define it using golang, what should I do？

**Comment by fffonion** [comment]
Hi @wangyufengx custom schema validator is not supported in Go, you will need to implement your own using like https://github.com/go-playground/validator.

**Reply by wangyufengx** [reply]
> Hi @wangyufengx custom schema validator is not supported in Go, you will need to implement your own using like https://github.com/go-playground/validator.

thank you ! I try it
